recipes/docx-mailmerge/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "docx-mailmerge" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/docx-mailmerge-{{ version }}.tar.gz
+  sha256: 6fbe353bd2f87dc6621d9add7bd04896181a469cc543502f1acc0c2444fd632a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python =2.7|>3.3
```

Conda-forge does not build anything <py36, so let's make this selector a bit simpler.

```suggestion
    - python =>3.6
```

---

Never knew, thanks for that!